arch/arm64/boot/dts/qcom/sdm660-xiaomi-lavender.dts

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright (c) 2020, The Linux Foundation. All rights reserved.
+ * Copyright (c) 2020, Aleksey Minnekhanov <alexey.min@gmail.com>
+ */
+
+/dts-v1/;
+
+#include "sdm660.dtsi"
+
+/ {
+	model = "Xiaomi Redmi Note 7";
+	compatible = "xiaomi,lavender", "qcom,sdm660";
+	/* required for bootloader to select correct board */
+	qcom,msm-id = <0x13d 0x0>;
+	qcom,board-id = <0x20008 0>;
+	qcom,pmic-id = <0x0001001b 0x0101011a 0x0 0x0>,
+			<0x0001001b 0x0201011a 0x0 0x0>,
+			<0x0001001b 0x0102001a 0x0 0x0>;
```

Comment:
Did you check if these are necessary?

Reply:
I did, it boots without all those IDs. They are probably necessary for DTBs in dtbo partition though.. I'm not going to check that now

Reply:
When booting twrp
```
Decompressing kernel image start: 3945 ms
Decompressing kernel image done: 4416 ms
Dtbo hdr magic mismatch 0, with D7B7AB1E
Best match DTB tags 317/00020008/0x00000000/0/1001B/102001A/0/0/(offset)0x979E72A8/(size)0x00084890
Using pmic info 0x1001B/0x102001A/0x0/0x0 for device 0x2001B/0x102001A/0x0/0x0
Memory Base Address: 0x40000000
smem protocol = 9C2FB0A0
```
When booting my image
```
Decompressing kernel image start: 2648 ms
Decompressing kernel image done: 2800 ms
Dtbo hdr magic mismatch 0, with D7B7AB1E
DTB offset is incorrect, kernel image does not have appended DTB
smem protocol = 9C2FF0A0
```

Reply:
IDs removed, can be booted without them